Build failed; the context from the [latest run](https://ci.jenkins.io/job/Infra/job/evergreen/job/PR-145/5/display/redirect) is:
<details><summary>Expand to view</summary>

```
[Docker Cloud image] Stopping distribution_backend_1  ... 
[Docker Cloud image] Stopping distribution_db_1       ... 
[Docker Cloud image] [3A[2K
Stopping distribution_instance_1 ... [32mdone[0m
[3B[2A[2K
Stopping distribution_backend_1  ... [32mdone[0m
[2B[1A[2K
Stopping distribution_db_1       ... [32mdone[0m
[1BRemoving distribution_instance_1 ... 
[Docker Cloud image] Removing distribution_backend_1  ... 
[Docker Cloud image] Removing distribution_db_1       ... 
[Docker Cloud image] [3A[2K
Removing distribution_instance_1 ... [32mdone[0m
[3B[2A[2K
Removing distribution_backend_1  ... [32mdone[0m
[2B[1A[2K
Removing distribution_db_1       ... [32mdone[0m
[1BRemoving network distribution_default
[Docker Cloud image] G'day!
[Docker Cloud image] 
[Docker Cloud image] Ran 16 tests.
[Docker Cloud image] ./shunit2/shunit2: line 950: [: skipped=3: integer expression expected
[Docker Cloud image] 
[Docker Cloud image] FAILED (failures=28)
[Docker Cloud image] Makefile:36: recipe for target 'docker-cloud-container-check' failed
[Docker Cloud image] make: *** [docker-cloud-container-check] Error 1
[Pipeline] [Docker Cloud image] }
[Pipeline] [Docker Cloud image] // dir
Post stage
[Pipeline] [Docker Cloud image] archiveArtifacts
[Docker Cloud image] Archiving artifacts
[Pipeline] [Docker Cloud image] }
[Pipeline] [Docker Cloud image] // withEnv
[Pipeline] [Docker Cloud image] }
[Pipeline] [Docker Cloud image] // node
[Pipeline] [Docker Cloud image] }
[Pipeline] [Docker Cloud image] // stage
[Pipeline] [Docker Cloud image] }
[Docker Cloud image] Failed in branch Docker Cloud image
[Pipeline] // parallel
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Publish jenkins/evergreen)
Stage "Publish jenkins/evergreen" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // timestamps
[Pipeline] }
[Pipeline] // timeout
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline

GitHub has been notified of this commit’s build result

ERROR: script returned exit code 2
Finished: FAILURE

```
</details>

[Powered by the Comment Logger](https://github.com/jenkins-infra/community-functions/tree/master/comment-logger)

--- a/distribution/client/package.json
+++ b/distribution/client/package.json
@@ -34,7 +34,7 @@
     "winston": "^2.4.3",
     "xmlrpc": "^1.3.2",
     "request-promise": "^4.2.2",
-    "promise-request-retry": "^1.0.0",
+    "promise-request-retry": "^1.0.1",
     "checksum": "^0.1.1"
   },
   "jest": {

